README.md

```diff
@@ -25,34 +37,41 @@ Example installation:
 
 `spack install dla-future ^intel-mkl`
 
-Notice that, for the package to work correctly, the HPX option `max_cpu_count` must be set accordingly to the platform,
-as it represents the maximum number of OS-threads.
+Notice that, for the package to work correctly, the HPX option `max_cpu_count` must be set accordingly to the architecture, as it represents the maximum number of OS-threads.
 
 `spack install dla-future ^intel-mkl ^hpx max_cpu_count=256`
 
-## How to use the library
+#### Build the old good way
 
-Using DLAF in a CMake project is extremely easy!
+You can build all the dependencies by yourself, but you have to ensure that:
+- BLAS/LAPACK implementation is not multithreaded
+- HPX: `HPX_WITH_NETWORKING=none` + `HPX_WITH_MAX_CPU_COUNT=n` (according to number of cores in the architecture, suggested the next closest power of 2)
+
```

Comment:
I would add that:

- HPX and DLAF need to have the same `CMAKE_BUILD_TYPE`, i.e. if HPX is built with `Debug`, then DLAF should also be built with `Debug`.

Reply:
I tried to give more details about how it works, I can revert and use your more straightforward one if you prefer. Let me know @rasolca @teonnik @msimberg